packages/commons-test/src/testUtils.ts

```diff
@@ -1296,3 +1297,12 @@ export const getMockUserNotificationConfig = (): UserNotificationConfig => ({
   createdAt: generateMock(z.coerce.date()),
   updatedAt: generateMock(z.coerce.date().optional()),
 });
+
+// eslint-disable-next-line @typescript-eslint/explicit-function-return-type
+export const expectTokenWith = (matchers: {
```

Comment:
moved this to commons-test (also refactoring it) so it can be reused in the bff and elsewhere